tests/roles/run_tests/templates/clusterctl-vars.yaml

```diff
@@ -15,13 +15,13 @@ CLUSTER_APIENDPOINT_HOST: {{ CLUSTER_APIENDPOINT_HOST }}
 CLUSTER_APIENDPOINT_PORT: {{ CLUSTER_APIENDPOINT_PORT }}
 POD_CIDR: {{ POD_CIDR }}
 SERVICE_CIDR: {{ SERVICE_CIDR }}
-PROVISIONING_POOL_RANGE_START: {{ PROVISIONING_POOL_RANGE_START }}
-PROVISIONING_POOL_RANGE_END: {{ PROVISIONING_POOL_RANGE_END }}
-PROVISIONING_CIDR: {{ PROVISIONING_CIDR }}
-BAREMETALV4_POOL_RANGE_START: {{ BAREMETALV4_POOL_RANGE_START }}
-BAREMETALV4_POOL_RANGE_END: {{ BAREMETALV4_POOL_RANGE_END }}
-BAREMETALV6_POOL_RANGE_START: {{ BAREMETALV6_POOL_RANGE_START }}
-BAREMETALV6_POOL_RANGE_END: {{ BAREMETALV6_POOL_RANGE_END }}
+PROVISIONING_POOL_RANGE_START: {{ IPAM_PROVISIONING_POOL_RANGE_START }}
+PROVISIONING_POOL_RANGE_END: {{ IPAM_PROVISIONING_POOL_RANGE_END }}
+BARE_METAL_PROVISIONER_CIDR: {{ BARE_METAL_PROVISIONER_CIDR }}
```

Comment:
Is the `BARE_METAL` vs `BAREMETAL` inconsistency something we want to have?

Reply:
done, i have changed every BAREMETAL to BARE_METAL.